src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/SelectShipping/_form.html.twig

```diff
@@ -20,7 +20,7 @@
                     </div>
                     <div class="extra">
                         <div class="ui large right floated fee label">
-                            {{ shipping_method_choice.parent.vars.shipping_costs[shipping_method_choice.parent.vars.choices[loop.index0].data.id]|sylius_price }}
+                            {{ shipping_method_choice.parent.vars.shipping_costs[shipping_method_choice.parent.vars.choices[loop.index0].value]|sylius_price }}
```

Comment:
This line looks very ugly, is there any way to replace it with a cleaner version?